www/class/centreonCustomView.class.php

```diff
@@ -1044,27 +1044,58 @@ public function getLayout($viewId)
     }
 
     /**
-     * @param $viewId
+     * @param int $viewId
      * @return array
      * @throws Exception
      */
-    public function getUsersFromViewId($viewId)
+    public function getUsersFromViewId(int $viewId): array
     {
         static $userList;
+        global $centreon;
 
         if (!isset($userList)) {
+            /**
+             * Get user's ACL
+             */
+            $arrayOfAclListOfContactIds = $centreon->user->access->getContactAclConf(
```

Comment:
I'm not sure it's a good practice to name variables like `arrayOf...`
eg, you could rename it as `allowedContactIds`